### ISSUE TITLE
Really a one-time purchase or nagware?

### ISSUE THREAD

**Issue by WolfgangDpunkt** [Suplanus/Cuterdio#151]
So far I have been very enthusiastic about the cuterdio ios app.
I also bought the theme editor to optimise the UI.
Unfortunately, I find the new "unlock cover" function quite distracting. I don't care if I see the cover or not (the placeholder I can set thanks to the theme editor is totally ok) but the purchase prompt is "ugly" and interferes with my radio enjoyment. But I can't hide the "Freischalten für immer" advertisement with the paid theme editor.

I understand very well that app development costs money, good work should be paid for. But I have already paid once, now another 5 euros so that the "nagscreen" disappears would also not be a big problem. But in how many days or weeks will there be a new banner, feature or something else that I have to "buy away"?

I don't mean this in a bad way, but the nagware principle bothers me a lot, so I think I'll switch to another gui for radio-browser.info 
But it's annoying that I have to pick my favourites again.
No offence, but if a paid app is "degraded" in the user experience afterwards to motivate another payment, then I lose trust.

**Comment by Suplanus** [comment]
Hi Wolfgang,

thanks for your words.
It was not easy for me to implement the paid cover unlock... I wrote a few words here: https://maui-blog.com/one-year-on-the-app-store-with-beer/

TLDR: My costs weren't covered... My motivation for Cuterdio was at the ground. So I made the decision.

I also thinking about if the one time purchase of the "Unlock cover" is enough for all features... But thats how I startet now.
It looks good that I can remove the purchase for ThemeEditor or other Features.  
For example:  
I am currently working on [Time based playing](https://github.com/Suplanus/Cuterdio/issues/92), and I don't know if it's OK for a separate in app purchase or not.
I am not a big fan of a subscription model for a small app like Cuterdio. So there will be no subscription.

I would like to deliver Cuterdio for free with beer tips, but it don't work. Now with the cover unlock it works. But in one year with a small user base I don't know if this is OK for me and the running costs.

There will be no existing feature which will be paid... the unlock cover was an exception.  
So the question is: Would the new feature will be OK for you that there is an extra in app purchase?

Thanks again for your opinion 💖

**Comment by WolfgangDpunkt** [comment]
Good day!

Thank you for your very kind and understanding reply.

I think we see things very similarly. Subscription fees would not be a viable solution. Buying more features is a good idea for me. Only the "advertising" for them should only appear once at the most (for users who have already paid for something in another way).

I just read that the banner only appears every 5 songs, but it is  there all the time when you listen to news radio (Deutschlandfunk, for example). I think because there are no songs there in that sense technically.
I think you should include and sell all kinds of feature ideas (time-based playing sounds exciting), but only discreetly inform the regular users about it, and don't include a non-hideable notice in an app that is used every day.

Now I wish you a merry holiday season. 

I'll gladly do that with the activation now. and listen to my favourite radio stations again like before. 👍

**Comment by Suplanus** [comment]
I also think we have the same idea how we would like to pay for an app 🦄  

There is no in app advertisement for ThemeEditor. There will also no in app ad for "Time based playing". Only the menu items.   
You will see how it works, can also config it. But to use it you have to pay.  

The banner will be shown every fifth change of the "song". In the case of "Deutschlandfunk" I think there is a quick change of the topics which are discussed...

I am happy that you are happy... And I will rethink the situation, again.

Have merry holidays, stay save!

**Comment by Suplanus** [comment]
Sidenote: Time based playing is currently in beta via Testflight. You can test it free until the release 💖
